build.sbt

```diff
@@ -15,8 +15,7 @@ addCommandAlias(
 
 addCommandAlias(
   name = "sortImports",
-  value = ";scalafixEnable;compile:scalafix SortImports;test:scalafix SortImports;multi-jvm:scalafix SortImports;" +
-    "CompileJdk9:scalafix SortImports;TestJdk9:scalafix SortImports;scalafmtAll;test:compile;multi-jvm:compile;reload")
+  value = ";scalafixEnable;compile:scalafix SortImports;test:scalafix SortImports;scalafmtAll")
```

Comment:
why is scalafmtAll needed in the alias? does the sorting change the format of the imports? Is that for the grouped imports?

Reply:
it sometimes change to `{ A }` to `{A}`